controllers/operands/kubevirtConsolePlugin_test.go

```diff
@@ -203,12 +203,25 @@ var _ = Describe("Kubevirt Console Plugin", func() {
 			req = commontestutils.NewReq(hco)
 		})
 
-		It("should create if not present", func() {
-			expectedResource, err := NewKvUIPluginDeplymnt(hco)
+		DescribeTable("should create if not present", func(appComponent hcoutil.AppComponent) {
+			var expectedResource *appsv1.Deployment
+			var err error
+			switch appComponent {
+			case hcoutil.AppComponentUIPlugin:
+				expectedResource, err = NewKvUIPluginDeployment(hco)
+			case hcoutil.AppComponentUIProxy:
+				expectedResource, err = NewKvUIProxyDeployment(hco)
+			}
 			Expect(err).ToNot(HaveOccurred())
 
 			cl := commontestutils.InitClient([]client.Object{})
-			handler, err := newKvUIPluginDeploymentHandler(logger, cl, commontestutils.GetScheme(), hco)
+			var handler []Operand
+			switch appComponent {
+			case hcoutil.AppComponentUIPlugin:
+				handler, err = newKvUIPluginDeploymentHandler(logger, cl, commontestutils.GetScheme(), hco)
+			case hcoutil.AppComponentUIProxy:
+				handler, err = newKvUIProxyDeploymentHandler(logger, cl, commontestutils.GetScheme(), hco)
+			}
 			Expect(err).ToNot(HaveOccurred())
```

Comment:
Please consider to pass the handler as a parameter. This will be possible if we'll get read of all the return values above. The result will be cleaner code, without code branches.